x-pack/plugins/fleet/common/settings/agent_policy_settings.ts

```diff
@@ -37,4 +37,93 @@ export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
     },
     schema: z.number().int().min(0).default(0),
   },
+  {
+    name: 'agent.download.timeout',
+    hidden: true,
+    title: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.downloadTimeoutTitle', {
+      defaultMessage: 'Agent binary download timeout',
+    }),
+    description: i18n.translate(
+      'xpack.fleet.settings.agentPolicyAdvanced.downloadTimeoutDescription',
+      {
+        defaultMessage: 'Timeout for downloading the agent binary',
+      }
+    ),
+    learnMoreLink:
+      'https://www.elastic.co/guide/en/fleet/current/enable-custom-policy-settings.html#configure-agent-download-timeout',
+    api_field: {
+      name: 'agent_download_timeout',
+    },
+    schema: zodStringWithDurationValidation.default('2h'),
+  },
+  {
+    name: 'agent.download.target_directory',
+    hidden: true,
+    api_field: {
+      name: 'agent_download_target_directory',
+    },
+    title: i18n.translate(
+      'xpack.fleet.settings.agentPolicyAdvanced.agentDownloadTargetDirectoryTitle',
+      {
+        defaultMessage: 'Agent binary target directory',
+      }
+    ),
+    description: i18n.translate(
+      'xpack.fleet.settings.agentPolicyAdvanced.agentDownloadTargetDirectoryDescription',
+      {
+        defaultMessage: 'The disk path to which the agent binary will be downloaded',
+      }
+    ),
+    learnMoreLink:
+      'https://www.elastic.co/guide/en/fleet/current/elastic-agent-standalone-download.html',
+    schema: z.string(),
+  },
+  {
+    name: 'agent.logging.metrics.period',
+    hidden: true,
+    api_field: {
+      name: 'agent_logging_metrics_period',
+    },
+    title: i18n.translate(
+      'xpack.fleet.settings.agentPolicyAdvanced.agentLoggingMetricsPeriodTitle',
+      {
+        defaultMessage: 'Agent logging metrics period',
+      }
+    ),
+    description: i18n.translate(
+      'xpack.fleet.settings.agentPolicyAdvanced.agentLoggingMetricsPeriodDescription',
+      {
+        defaultMessage: 'The frequency of agent metrics logging',
+      }
+    ),
+    learnMoreLink:
+      'https://www.elastic.co/guide/en/fleet/current/elastic-agent-standalone-logging-config.html#elastic-agent-standalone-logging-settings',
+    schema: zodStringWithDurationValidation.default('30s'),
+  },
+  {
+    name: 'agent.monitoring.http',
+    hidden: true,
```

Comment:
Added a `hidden` flag to be able to hide a config without deleting from this file. This acts like a feature flag, so when a setting is ready to be rendered, just update `hidden: false`.